src/util/mod.rs

```diff
@@ -62,4 +65,63 @@ pub(crate) mod tests {
 
         assert_abs_diff_eq!(a, b, epsilon = epsilon);
     }
+
+    pub trait PseudoRandom {
+        fn pseudo_random(len: usize, device: &Device) -> Self;
+    }
+
+    impl PseudoRandom for Tensor {
+        fn pseudo_random(len: usize, device: &Device) -> Self {
+            let mut rng = Pcg32::new(len as u64, 0);
```

Comment:
Let's add a comment explaining why we don't want to use `rand`'s built-in `Uniform` sampler.